MSAL/src/MSALDeviceInformation.m

```diff
@@ -54,14 +54,7 @@ - (instancetype)init
         _deviceMode = MSALDeviceModeDefault;
         _extraDeviceInformation = [NSMutableDictionary new];
         
-        if (@available(macOS 10.15, *))
```

Comment:
Minimum version for macOS is 10.15 (101500) so all checks like this have been removed as they are now always true